src/arrayMethodJoin.js

```diff
@@ -4,9 +4,29 @@
  * Implement method join
  */
 function applyCustomJoin() {
-  [].__proto__.join2 = function(separator) {
+  [].__proto__.join2 = function(separator = ',') {
     // write code here
+    let result = '';
```

Comment:
Better change variable name for something more informative.

Reply:
Better create more meaningful variables. For example `stringFromArray` instead of  `result`.